Dockerfile.repo

```diff
@@ -57,31 +59,28 @@ RUN \
 USER root
 
 RUN \
-  yum install -y MTA mod_ssl /usr/sbin/semanage && \
-  cd /home/builder/rapache-1.2.9 && \
+  dnf install -y MTA mod_ssl /usr/sbin/semanage && \
+  cd /home/builder/rapache-master && \
   make install
 
   
 RUN \
   chkconfig httpd off \
   userdel builder
 
-RUN yum install -y centos-release-scl-rh && \
-    yum install -y llvm-toolset-7-clang && \
-    yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm && \
-    yum install -y llvm5.0-devel postgresql13-devel curl-devel libxml2-devel
-
 # NODE
 ENV NPM_CONFIG_LOGLEVEL warn
-ENV NODE_VERSION 14.15.1
-
-RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" \
-	&& tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
-	&& rm -f "node-v$NODE_VERSION-linux-x64.tar.gz" \
-  && npm install -g npm@latest \
-  && npm install -g coffeescript properties underscore underscore-deep-extend properties-parser flat glob dotenv dotenv-expand
+ENV NODE_VERSION 14.x
+RUN curl -fsSL https://rpm.nodesource.com/setup_$NODE_VERSION | bash - && \
```

Comment:
previously this was arch dependent binary curl.  This script is an easy way to install arch independent nodejs 14 repository and then subsequenetly install nodejs and npm.